src/modules/gdgem/srs/CMakeLists.txt

```diff
@@ -31,11 +31,12 @@ create_test_executable(ParserVMM3Test)
 set(VMM3CalibrationFileTest_SRC
   CalibrationFile.cpp
   CalibrationFileTest.cpp
-  ../../test/SaveBuffer.cpp
+  ${SD}/
```

Comment:
maybe SD is a little short here, what does it stand for?

Reply:
that is an oversight, I have changed the name to ESS_SOURCE_DIR elswere